binrz/man/rizin.1

```diff
@@ -1,132 +1,122 @@
+.It =
+Equivalent of 'rizin malloc://512'.
+.It -
+read file from stdin.
+.It -=
+perform R=! command to run all commands remotely
 .It Fl 0
-Print \\x00 after initialization and after every command executed
+Print \\x00 after initialization and after every command executed.
 .It Fl 2
-Close stderr before starting RzCore
-.It Fl a Ar arch
-force asm.arch (x86, ppc, arm, mips, bf, java, ...)
+close stderr file descriptor (silent warning messages).
+.It Fl a Ar [arch]
+Set the assembly architecture.(x86, ppc, arm, mips, bf, java, ...).
 .It Fl A
-run 'aaa' command before prompt or patch to analyze all referenced code. Use -AA to run aaaa
-.It Fl b Ar bits
-force asm.bits (16, 32, 64)
-.It Fl B Ar baddr
-Specify the base address to be used when loading a new binary. See 'e?bin.baddr'
-.It Fl c Ar cmd
-Execute the given command before giving prompt
+run 'aaa' command before prompt or patch to analyze all referenced code.
+.It Fl b Ar [bits]
+Set the assembly bitness. (16, 32, 64).
+.It Fl B Ar [baddr]
+Set the base address for Position-Independent Executables (PIE).
+.It Fl c Ar 'cmd..'
+Execute the specified Rizin command.
+.It -C
+Specifies a file in the format "host:port" to run Rizin remotely through HTTP.
 .It Fl d
-Start in debugger mode
-.It Fl D Ar dbg.backend
-Enable debug mode. Set cfg.debug=true
+Debug the executable 'file' or running process 'pid'.
+.It Fl D Ar [backend]
+Enable debug mode, setting the configuration variable 'cfg.debug=true'.
 .It Fl e Ar k=v
-Set configuration eval variable key=value. For example \-e scr.color=false
+Evaluate the specified configuration variable.
 .It Fl f
-Blocksize = file size
-.It Fl i Ar file
-Run script file. After the file is loaded
-.It Fl I Ar file
-Run script file. Before the file is loaded
-.It Fl k Ar kernel
-Select kernel (asm.os) for syscall resolution
-.It Fl l Ar plugfile
-Load given plugin file
+Set block size to be equal to file size.
+.It -F [binplugin]
+Forces the use of a specific rbin plugin.
+.It Fl h, hh
+Shows help messages, with -hh showing detailed help.
+.It Fl H ([var])
+display variable.
+.It Fl i Ar [file]
+Run the commands from the script file.
+.It Fl I Ar [file]
+Run script file. Before the file is loaded.
+.It Fl k Ar [OS/kern]
+Set the assembly operating system.
+.It Fl l Ar [lib]
+Load the specified plugin file.
 .It Fl L
 List supported IO plugins.
-.It Fl m Ar addr
-map file at given address
+.It Fl m Ar [addr]
+map file at given address.
 .It Fl M
-Disable demangling
-.It Fl n
-Do not perform any analysis (r_bin). Just load the raw file
-.It Fl nn
-Only load the rbin structures (elf, mach0, ...)
+Disable demangling.
+.It Fl n, nn
+Controls loading of RzBin info, with -nn only loading binary structures.
 .It Fl N
 Do not load user settings/projects from ~/.rizinrc, ~/.config/rizin/rizinrc and the scripts inside .../rizinrc.d/ directory.
 .It Fl NN
-Same as -N but also disables the automatic loading of plugins on startup time
+Disables loading any scripts or plugins.
```

Comment:
Please be consistent. Sometimes it `disable`, sometimes it `disables`. I suggest follow the same style everywhere.